processes/ingest/doab.py

```diff
@@ -30,142 +27,57 @@ class DOABProcess(CoreProcess):
     def __init__(self, *args):
         super(DOABProcess, self).__init__(*args[:4])
 
-        self.ingestOffset = int(args[5]) if args[5] else 0
-        self.ingestLimit = (int(args[4]) + self.ingestOffset) if args[4] else 10000
-
-        self.generateEngine()
-        self.createSession()
+        self.offset = int(args[5]) if args[5] else 0
+        self.limit = (int(args[4]) + self.offset) if args[4] else 10000
 
-        self.s3_manager = S3Manager()
-        self.s3_manager.createS3Client()
-        self.s3Bucket = os.environ['FILE_BUCKET']
+        self.dSpace_service = DSpaceService(
+            base_url=self.DOAB_BASE_URL, SourceMapping=DOABMapping)
 
-        self.fileQueue = os.environ['FILE_QUEUE']
-        self.fileRoute = os.environ['FILE_ROUTING_KEY']
+    def runProcess(self):
+        try:
+            self.generateEngine()
+            self.createSession()
 
-        self.rabbitmq_manager = RabbitMQManager()
-        self.rabbitmq_manager.createRabbitConnection()
-        self.rabbitmq_manager.createOrConnectQueue(self.fileQueue, self.fileRoute)
+            records = []
 
-        self.constants = get_constants()
+            if self.process == 'daily':
+                records = self.dSpace_service.get_records(offset=self.offset, limit=self.limit)
+            elif self.process == 'complete':
+                records = self.dSpace_service.get_records(full_or_partial=True, offset=self.offset, limit=self.limit)
```

Comment:
I recommend changing the name `full_or_partial` to `full_import` since it's more clear what this variable does and the publisherBacklistProcess uses that variable name too.

Reply:
Following up you could also just remove the full param!